src/internal/ob.js

```diff
@@ -14,19 +14,31 @@ const ux_ffff_ffff_0000_0000 = new BN('ffffffff00000000', 'hex')
 const u_65535 = new BN('65535')
 const u_65536 = new BN('65536')
```

Comment:
Any reason we're using these instead of their hex-named equivalents? Seems like using hex notation would be both more consistent and more readable.